### PR TITLE
Graduate installer-help redesign experiment into prod.

### PR DESCRIPTION
**Pull request by amychurchwell**
## Description
Graduates the installer-help redesign experiment! 🎉  ([Original PR](https://github.com/mozilla/bedrock/pull/8731))

**Experiment Analysis Report:** https://docs.google.com/document/d/1RlsO0zjuZT2r717TOdfv8PpOGBfvlVHu3SO5-cpk5F4/edit# 

## Issue
[Airtable card](https://airtable.com/tblO5zD4uNk7ISyG0/viwIUJsHKDrOFl90B/reclbibhH3nIMzdyp)

### CODE REVIEW

File: bedrock/firefox/templates/firefox/installer-help.html [hunk 1]
```diff
@@ -20,56 +20,97 @@
...
-    </header>
-    <aside class="mzp-c-emphasis-box">
-      <h3 class="installer-help-emphasis-title">{{ _('Need help installing?') }}</h3>
-      <p class="installer-help-emphasis-tagline"><a href="https://support.mozilla.org/kb/install-firefox-windows">{{ _('Visit support.mozilla.org') }}</a></p>
-    </aside>
-  </div>
+<main class="mzp-l-content">
+  <section class="mzp-c-hero">
+    <div class="mzp-c-hero-body">
+      <h1 class="mzp-c-hero-title">{{ _('Hmm… your download was interrupted') }}</h1>
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/installer-help.html:37]: Should this new design template be behind an L10 conditional in the view? Currently this PR is losing all existing translations, unless I'm misreading something?

Comment (amychurchwell) [reply bedrock/firefox/templates/firefox/installer-help.html:37]: you are correct! I jumped the gun.

File: bedrock/firefox/tests/test_base.py [hunk 2]
```diff
@@ -84,7 +94,9 @@ def test_one_button_when_channel_specified(self):
         self.client.get(self.url, {
             'channel': 'beta',
         })
-        self.button_mock.assert_called_once_with('beta', force_direct=True,
+        self.button_mock.assert_called_once_with('beta',
+                                                 alt_copy=Markup('Download Now'), button_color='mzp-t-small',
```
Comment (alexgibson) [inline bedrock/firefox/tests/test_base.py:98]: I like the way this test checks that the button is primary only if there's a single one on the page 👍

File: bedrock/firefox/views.py [hunk 3]
```diff
@@ -81,8 +77,10 @@ def installer_help(request):
         else:
             context['installer_channel'] = installer_channel
 
-    if variant == 'b':
-        template = 'firefox/installer-help-b.html'
+    if lang_file_is_active('firefox/installer-help-redesign', locale):
```
Comment (alexgibson) [inline bedrock/firefox/views.py:80]: Suggest setting this .lang file explicitly in the template using `{% add_lang_files "firefox/installer-help-redesign" %}`. This way if the template location ever moves, the translations don't break.

**Comment by alexgibson** [comment]
Translations for download buttons are now supplied by Fluent, so in the new template here I'd suggest using Fluent ID's instead of .lang strings.

So instead of:

```
alt_copy=_('Download Now')
```

use:

```
alt_copy=ftl('download-button-download-now')
```

**Pull request merged**